documentation/modules/serving/pages/knative-client.adoc

```diff
@@ -91,7 +90,7 @@ You can list the created services using the command:
 [.console-input]
 [source,bash,subs="+macros,attributes+"]
 ----
-kn service list --namespace {tutorial-namespace}
+kn service list
```

Comment:
Commands in this page don't have `--namespace` in them. I removed 1-2 I saw.

Reply:
I think its better to add namespace everywhere for consistency with other chapters ? Can you fix that ?